opentech/apply/projects/templates/application_projects/project_simplified_detail.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base-apply.html" %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.submission.page }}</span>
+            <span>{{ object.submission.round }}</span>
+            <span>Lead: {{ object.lead }}</span>
+        </h5>
+    </div>
+</div>
+
+<div class="wrapper wrapper--large">
+
+    {% block notifications %}
+    {% endblock %}
+    <div class="wrapper wrapper--sidebar">
+        <article class="wrapper--sidebar--inner">
+            <h3>Project Information</h3>
+            <div class="grid grid--proposal-info">
+                <div>
+                    <h5>Proposed start date</h5>
+                    <p>{{ object.proposed_start|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Project Proposed end date</h5>
+                    <p>{{ object.proposed_end|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Legal name</h5>
+                    <p>{{ object.contact_legal_name|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Email</h5>
+                    {% if object.contact_email %}
+                    <a href="mailto:{{ object.contact_email }}">{{ object.contact_email }}</a>
+                    {% else %}
+                    -
+                    {% endif %}
+                </div>
+
+                <div>
+                    <h5>Address</h5>
+                    <p>{{ object.get_address_display|default:"-"}}</p>
+                </div>
+
+                <div>
+                    <h5>Phone</h5>
+                    <p>{{ object.phone|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Value</h5>
+                    <p>${{ object.value|default:"-" }}</p>
+                </div>
+            </div>
+
+            <hr />
+
+            <p>TODO: review info goes here</p>
+        </article>
+
+        <aside class="sidebar">
+
+            <div class="sidebar__inner">
+                <h5>Related Pages</h5>
+
+                <p><a href="{{ object.get_absolute_url }}">Project Page</a></p>
```

Comment:
This should be a breadcrumb at the top of the page.

similar to the "projects" link below - chris to style:
![image](https://user-images.githubusercontent.com/6338661/63498980-d81b2a00-c4be-11e9-8998-d7c3dc54d72d.png)

Reply:
Fixed and rebased.